### ISSUE TITLE
Does the new item initial modal make sense?

### ISSUE THREAD

**Issue by webteckie** [keystonejs/keystone#2201]
Why display an initial modal instead of just displaying the entire form?  I have a situation where I have a bunch of list fields that are required and doing this in the initial modal looks a bit ugly.  Can we do away with initial and just display the full form?

**Comment by jossmac** [comment]
This is something @twalve and I have been discussing recently, for a current project. Perhaps it could be a feature of `autocreate: true`?

**Comment by webteckie** [comment]
Never seen that feature but nevertheless it doesn't sound like a natural fit here.  What does or would it do?  I don't think autocreating will solve it since the user must still input the required values.

**Comment by webteckie** [comment]
@JedWatson does the `autocreate: true` address this?  I have not used that feature and do not see documented either.

**Comment by JedWatson** [comment]
`autocreate: true` is really a different feature, I think. It's for when you can create a new item without any user input (e.g. all required values can be generated)

The create modal exists because not all form fields can be used without an existing item... the form works quite differently in create / edit modes. Might be something we can address down the track though.

**Comment by webteckie** [comment]
Ok moved to backlog for now.

**Comment by creynders** [comment]
@JedWatson I really think we should move towards dropping the initial modal. It's not so great UX, IMO. And surely the form fields function with a throwaway mongoose document, no? I.e. a `new Model` w/o saving it to the DB. Or are there fields that look up the old version of the document or something?

**Comment by mxstbr** [comment]
Now (i.e. after #2566 lands) would be the perfect opportunity to drop this. Do we want to do that @JedWatson?

**Comment by creynders** [comment]
💯 👍 @JedWatson @mxstbr

**Comment by dlindenkreuz** [comment]
Any progress on this?

**Comment by gautamsi** [comment]
closing in favor of #1840.
given that Keystone 4 is going in maintenance mode (see #4913 for details). This may or may not land in 4.x.
